docs/app-dev-context/custom-logging.md

```diff
@@ -0,0 +1 @@
+Use a custom logger for logging actions of the Temporal Client.
```

Comment:
Let's just call it "logging".

If you want to log from a Workflow - in Go for example you should use the SDK logger, so that the logs are not constantly repeated.

Activities it matters less.

- How to log from a Workflow
- How to use a custom Logger